Dockerfile.aro-multistage

```diff
@@ -0,0 +1,22 @@
+# Uses a multi-stage container build to build the RP.
+#
+# TODO:
+# Currently the docker version on our RHEL7 VMSS uses a version which
+# does not support multi-stage builds.  This is a temporary stop-gap
+# until we get podman working without issue
+FROM registry.access.redhat.com/ubi7/go-toolset:1.14.12 AS builder
+ENV GOOS=linux \
+    GOPATH=/go/
+WORKDIR ${GOPATH}/src/github.com/Azure/ARO-RP
+USER root
+RUN yum update -y && \
+    yum install libassuan-devel openssl -y
```

Comment:
what are the reasons either of these are needed?  Both surprise me.

Reply:
Not needed - removed